TGM/src/main/java/network/warzone/tgm/parser/banner/BannerPatternsDeserializer.java

```diff
@@ -6,6 +6,7 @@
 import org.bukkit.DyeColor;
 import org.bukkit.block.banner.Pattern;
 import org.bukkit.block.banner.PatternType;
+import org.bukkit.craftbukkit.libs.org.apache.commons.lang3.EnumUtils;
```

Comment:
This is using a craftbukkit shaded library, please use `org.apache.commons.lang3.EnumUtils` instead